clang/include/clang/Driver/Options.td

```diff
@@ -1822,6 +1822,10 @@ def fsycl_add_targets_EQ : CommaJoined<["-"], "fsycl-add-targets=">, Flags<[Driv
   HelpText<"Specify comma-separated list of triple and device binary image pairs to add to the final SYCL binary">;
 def fsycl_link_targets_EQ : CommaJoined<["-"], "fsycl-link-targets=">, Flags<[DriverOption, CC1Option, CoreOption]>,
   HelpText<"Specify comma-separated list of triples SYCL offloading targets to produce linked device images">;
+def fsycl_device_code_split : Flag<["-"], "fsycl-device-code-split">,
+  Flags<[CC1Option, CoreOption]>, HelpText<"Perform SYCL device code split">;
+def fsycl_one_kernel_per_module : Flag<["-"], "fsycl-one-kernel-per-module">,
+  Flags<[CC1Option, CoreOption]>, HelpText<"Emit a separate device module for each kernel">;
```

Comment:
(overlooked this in the previous review)
I think it would be better to have a single option with multiple possible values instead of two (or potentially more if other groupings are invented). E.g.
-fsycl-device-code-split=per_kernel
-fsycl-device-code-split=per_source

Reply:
Done.